core/src/main/scala/org/apache/spark/deploy/history/EventLogFileWriters.scala

```diff
@@ -56,7 +56,7 @@ abstract class EventLogFileWriter(
 
   protected val shouldCompress = sparkConf.get(EVENT_LOG_COMPRESS)
   protected val shouldOverwrite = sparkConf.get(EVENT_LOG_OVERWRITE)
-  protected val outputBufferSize = sparkConf.get(EVENT_LOG_OUTPUT_BUFFER_SIZE).toInt
+  protected val outputBufferSize = sparkConf.get(EVENT_LOG_OUTPUT_BUFFER_SIZE).toInt * 1024
```

Comment:
Let's file a JIRA. this will also change the default buffer size being used, and I think we should track this bug via JIRA

Reply:
Thanks for your reply, I have filed a JIRA https://issues.apache.org/jira/browse/SPARK-45333